torch/csrc/autograd/VariableTypeUtils.h

```diff
@@ -182,6 +184,7 @@ inline Tensor as_view(const Tensor & base, const Tensor & tensor, bool is_bw_dif
       auto diff_view_meta = static_cast<DifferentiableViewMeta*>(base_meta);
       const auto& base_fw_info = diff_view_meta->get_forward_view();
       new_fw_info = base_fw_info.chain(base, tensor, view_func);
+      creation_meta = propagate_creation_meta(diff_view_meta->get_creation_meta(), creation_meta);
```

Comment:
I don't think you need to replicate this and l 170.
You can just add this below just next to the call to `make_variable_differentiable_view`.